src/mongoModelsRepo/overwritePersonaAttribute.ts

```diff
@@ -23,19 +21,24 @@ const overwritePersonaAttribute = (config: Config) => {
     try {
       const result = await collection.findOneAndUpdate({
         key,
-        organisation: new ObjectID(organisation),
-        personaId: new ObjectID(personaId),
+        organisation: new ObjectId(organisation),
+        personaId: new ObjectId(personaId),
       },
       {
         $set: {
           value,
         },
       },
       {
-        returnOriginal: false,
+        returnDocument: 'after',
```

Comment:
I think we should use the enum from `mongodb` here:
```suggestion
        returnDocument: ReturnDocument.AFTER,
```